pkg/options/options.go

```diff
@@ -29,6 +31,7 @@ var env struct {
 	EnableClientHandlingTimeHistogram      bool `envconfig:"ENABLE_CLIENT_HANDLING_TIME_HISTOGRAM" default:"true"`
 	EnableClientStreamReceiveTimeHistogram bool `envconfig:"ENABLE_CLIENT_STREAM_RECEIVE_TIME_HISTOGRAM" default:"true"`
 	EnableClientStreamSendTimeHistogram    bool `envconfig:"ENABLE_CLIENT_STREAM_SEND_TIME_HISTOGRAM" default:"true"`
+	GrpcClientMaxRetry                     int  `envconfig:"GRPC_CLIENT_MAX_RETRY" default:"5"`
```

Comment:
I am not familiar with customization option for envconfig, but can a CLI specify its own config here?

Reply:
Also, IIUC @mattmoor suggests default to be 0 here (WithMax(0) will disable retry according to the doc)

Reply:
if they can set an env var, they can set this.

not sure how that works on windows in cli though

Reply:
changed to default to 0